az-core/src/main/java/azkaban/Constants.java

```diff
@@ -161,6 +161,8 @@
 
     // enable Quartz Scheduler if true.
     public static final String ENABLE_QUARTZ= "azkaban.server.schedule.enable_quartz";
+
+    public static final String CUSTOM_CREDENTIAL_NAME = "azkaban.security.credential";
```

Comment:
Where would this be configured? In azkaban.properties?

Reply:
No, this will be configured under jobtype/commonprivate.properties.